examples/tests/valid/subdir2/file1.w

```diff
@@ -0,0 +1,11 @@
+bring "./inner" as blah; // an alias that is not "inner"
+
+class Foo {
+  pub foo(): str {
+    return "foo";
+  }
+
+  pub checkWidget(widget: blah.Widget): num {
```

Comment:
Perhaps add a static call to `bla.Widget.someStaticMethod()` because this example only references a type from `bla` without any runtime manifestation.

Reply:
Added